randovania/game_description/pickup/pickup_category.py

```diff
@@ -54,7 +54,7 @@ def is_expansion(self) -> bool:
 
 
 USELESS_PICKUP_CATEGORY = PickupCategory(
-    name="useless", long_name="Useless", hint_details=("an ", "Energy Transfer Module"), hinted_as_major=False
+    name="useless", long_name="Useless", hint_details=("a ", "useless item"), hinted_as_major=False
```

Comment:
this loses a lot of specificity for CS because CS already has a category with these details which was treated as separate from the Nothing hint category

Reply:
oh, wasn't aware of that. So CS should instead have another text for the hint details instead, so that "a useless item" can keep being used for what its used for in CS right now, and hten the other one for the nothing/empty items?